src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -10,6 +10,8 @@
 #include "ascii.hpp"
 #include "../../types/inc/utils.hpp"
 
+#include <Shlwapi.h>

```

Comment:
Oh boy I'm not sure we can use this header here. I'm not sure shlwapi can be used in onecore. This doesn't matter so much in the OSS build, but this'll break when it's ingested by the OS build system. (@miniksa sanity check me on this)

Reply:
We can, I believe. It's not really about the header, it's about which API is called. Presuming this was for `UrlUnescapeW` (through the macro `UrlUnescapeInPlace`)... then that's in `kernelbase.dll` and projected though `onecore.lib` so it's present in all editions.

Reply:
Okay thanks for the sanity check. I immediately saw `Shl`, thought "shell", and presumed it was a Desktop-only thing.